packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -49,6 +49,7 @@ const BLOCK_EDITOR_SETTINGS = [
 	'focusMode',
 	'distractionFree',
 	'fontSizes',
+	'formOptions',
```

Comment:
This only exists if the experiment is enabled. Can we condition it?

Reply:
This was implemented to allow 3rd-party plugins to add their own methods.
I removed it for now, I think we can do without it in the 1st implementation of the experiment, and add it later if needed 👍